cmd/tasks/src/lib.rs

```diff
@@ -200,18 +200,25 @@ fn tasks(context: &mut humility::ExecutionContext) -> Result<()> {
     loop {
         core.halt()?;
 
-        let cur =
-            core.read_word_32(hubris.lookup_symword("CURRENT_TASK_PTR")?)?;
+        let cur = hubris.current_task(core)?;
+        let dump_task = hubris.dump_task();
```

Comment:
Same here – a comment explaining that this is a special case for dumps with only a single task would make this more obvious.  From the variable name, I was expecting this to be handling Hubris archives with a `dump-agent` task in them, which didn't make much sense.